test/vm/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ set(UBUNTU_CONTAINER1 "vm-ubuntu2004:latest")
 set(UBUNTU_CONTAINER2 "vm-ubuntu2204:latest")
 set(UBUNTU_CONTAINER3 "vm-ubuntu2310:latest")
 set(DEBIANBUSTER_CONTAINER "vm-debianbuster:latest")
+# TODO In case we have an equivalent upstream supported container, remove the custom modern-falco-builder
+set(MODERN_FALCO_BUILDER_CONTAINER "modern-falco-builder:latest")
```

Comment:
Unless I am missing something, but the container defined in the Falco repo is specifically setup to build Falco in our CI https://github.com/falcosecurity/falco/blob/master/docker/builder/modern-falco-builder.Dockerfile

The previous falco-builder container setup appears to have been deprecated.

Reply:
yes, it is specific for Falco, but we don't use it anymore so probably it will be deprecated at the same say...it's ok to create a new one